pgo/workloads/cs-example/scylla.yaml

```diff
@@ -0,0 +1,530 @@
+# Scylla storage config YAML
+
+#######################################
+# This file is split to two sections:
+# 1. Supported parameters
+# 2. Unsupported parameters: reserved for future use or backwards
+#    compatibility.
+# Scylla will only read and use the first segment
+#######################################
+
+### Supported Parameters
```

Comment:
Better to derive this from the regular scylla.yaml so we don't have to maintain yet another config file.